RSDKv3/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+# pkg-config is inconvenient on Windows, perhaps we could use vcpkg there?
```

Comment:
We can very much use VCPKG. We already have VCPKG .props files ready to use in `/props` that I made previously to get GitHub CI working that can probably be condensed into 1 and tweaked to manage CMake as well. Though, for now, this is fine, and you can keep it if you don't wanna go through the extra work, just make sure you add it to the README as well.

Reply:
Cool. I've seen a few projects that use both pkg-config and vcpkg through CMake.
I'll give it a try on a Windows PC and let you know how it goes.